frontend/src/component/menu/Header/Header.tsx

```diff
@@ -132,12 +132,23 @@ const Header: VFC = () => {
     const routes = getRoutes();
 
     const filterByMode = (route: INavigationMenuItem): boolean => {
-        const { mode } = route.menu;
-        return (
-            !mode ||
-            (mode.includes('pro') && isPro()) ||
-            (mode.includes('enterprise') && isEnterprise())
-        );
+        const { mode, showEnterpriseBadge } = route.menu;
+
+        if (!mode) return true;
+
+        if (isPro()) {
+            return (
+                mode.includes('pro') ||
+                (mode.includes('enterprise') && showEnterpriseBadge) ||
```

Comment:
what exactly is `mode`? what does it mean for mode to include enterprise && showEnterpriseBadge? Is showEnterpriseBadge only true on pro subscriptions? Is it equivalent to `isEnterprise`?

Reply:
`showEnterpriseBadge` is a flag

Reply:
I'll simplify this. It decoupled showing the badge from showing the route. I don't think we need it